monai/networks/trt_compiler.py

```
@@ -261,7 +261,7 @@ def parse_groups(
        Tuple of Union[torch.Tensor, List[torch.Tensor]], according to the grouping in output_lists
 
     """
-    groups: tuple[torch.Tensor | list[torch.Tensor], ...] = tuple()
+    groups: tuple[torch.Tensor | list[torch.Tensor], ...] = ()
     cur = 0
     for l in range(len(output_lists)):
```

Ruff (0.15.1)

[error] Ambiguous variable name: `l`

(E741)